java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/accessor/impl/numeric/ArrowFlightJdbcBaseIntVectorAccessorUnitTest.java

```diff
@@ -62,35 +64,34 @@
         };
         if (vector instanceof UInt1Vector) {
           return new ArrowFlightJdbcBaseIntVectorAccessor((UInt1Vector) vector, getCurrentRow,
-              noOpWasNullConsumer);
+          noOpWasNullConsumer);
```

Comment:
Wrong formatting?

Reply:
Also checking this format on the rest of the methods in this if statement

Reply:
fixed

Reply:
Not fixed yet